language/documentation/hackathon-tutorial/step_1/BasicCoin/Move.toml

```diff
@@ -0,0 +1,6 @@
+[package]
+name = "BasicCoin"
+version = "0.0.0"
+
+[addresses]
+NamedAddr = "0xDEADBEEF"
```

Comment:
Maybe this name is a bit too dark ? ;-) What about 0x8888 or something more lucky?